src/zmalloc.h

```diff
@@ -62,11 +62,18 @@
 #endif
 
 /* On native libc implementations, we should still do our best to provide a
- * HAVE_MALLOC_SIZE capability.
+ * HAVE_MALLOC_SIZE capability. This can be set explicitly as well:
+ *
+ * NO_MALLOC_USABLE_SIZE disables it on all platforms, even if they are
+ *      known to support it.
+ * USE_MALLOC_USABLE_SIZE forces use of malloc_usable_size() regardless
+ *      of platform.
  */
 #ifndef ZMALLOC_LIB
 #define ZMALLOC_LIB "libc"
-#if defined(__GLIBC__) || defined(__FreeBSD__)
+#if !defined(NO_MALLOC_USABLE_SIZE) && \
+    (defined(__GLIBC__) || defined(__FreeBSD__) || \
+     defined(USE_MALLOC_USABLE_SIZE))
```

Comment:
so musl does support malloc_size...
why not support it implicitly (without requiring the USE_MALLOC_USABLE_SIZE macro? (in which case we can remove it from daily.yaml)

Reply:
It appears there's no reliable way to detect it (`musl` or explicitly existence of `malloc_usable_size`) based on header files (`autoconf` would do it in a breeze).

Reply:
musl can't be detected with an `ifdef` like we detect libc? or doe it not always support malloc_size?

Reply:
It can't.